client/src/root/Footer.jsx

```diff
@@ -139,15 +133,13 @@ class Footer extends React.Component {
             {' | '}
             <div style={styleImageAttribution}>
               <FooterLink href={`${this.props.homeUrl}about#attribution`}>
```

Comment:
Now that the footer attribution link is an indirect reference to another external link on our "About Us" page, we should get rid of the "You're about to leave the government blah blah..." warning if it's not actually navigating outside our client and it's just going to prompt them twice.

Reply:
That only happens on localhost. It's just reusing the custom link that checks, but it should work correctly (I think) in production?